### PR TITLE
Add logo and a bit more description to readthedocs

### PR DESCRIPTION
**Pull request by hyanwong**

### CODE REVIEW

**Comment by hyanwong** [comment]
I changed the logo to one with pale text.

Also note that I had problems compiling the docs because of unmatched doxygen lines in `c/tskit/tables.h` like `/** @} */` (e.g. [here](https://github.com/tskit-dev/tskit/blob/066c3d0f399510074f2ef2a7044ebe2351842aa8/c/tskit/tables.h#L1220). I'm not sure why my doxygen version complains but others do not, but it would be helpful to me if I could remove them.

**Comment by petrelharp** [comment]
Very nice, thanks for this. The docs compile fine for me; no idea why yours don't? My doxygen version is 1.8.13...

**Comment by hyanwong** [comment]
Thanks @petrelharp - my doxygen is 1.8.16. I'm not sure what the unmatched doxygen braces are there for - I'll ask Jerome.

File: docs/introduction.rst [hunk 1]
```diff
@@ -4,8 +4,19 @@
 Introduction
 ============
 
-This is the documentation for tskit, the tree sequence toolkit.
+This is the documentation for tskit, the tree sequence toolkit. Tree sequences
+are a highly efficient way of storing a set of related DNA sequences by storing
```
Comment (jeromekelleher) [inline docs/introduction.rst]: Little clunky using 'storing' twice in this sentence. Perhaps use 'encoding' for one of these?

**Comment by jeromekelleher** [comment]
> Thanks @petrelharp - my doxygen is 1.8.16. I'm not sure what the unmatched doxygen braces are there for - I'll ask Jerome.

These look like errors to me. It'd be nice to check for this in CI.

**Comment by hyanwong** [comment]
> These look like errors to me. It'd be nice to check for this in CI.

Agree. I'm not sure how to do this, given that CI currently passes. I can't find a "strict" mode for doxygen.

The corrections to make matching braces are in https://github.com/tskit-dev/tskit/pull/310

**Comment by jeromekelleher** [comment]
Great, thanks @hyanwong.

**Pull request merged**